app/components/primer/alpha/select_panel_element.ts

```diff
@@ -466,6 +466,7 @@ export class SelectPanelElement extends HTMLElement {
     if (event.target === this.dialog && event.type === 'close') {
       // Remove data-ready so it can be set the next time the panel is opened
       this.dialog.removeAttribute('data-ready')
+      this.invokerElement?.setAttribute('aria-expanded', 'false')
```

Comment:
Can we move this to line `56` in the `close` eventListener?

Reply:
FYI, I just tried this out and for some reason the state doesn't correctly change back to being `aria-expanded=false` if I use Esc to close the menu

Reply:
maybe we should put it in both places. I am thinking that if a consumer closes the panel outside of the component this call might be missed.